common-money.lic

```diff
@@ -137,7 +137,27 @@ module DRCM
       return
     end
     minimize_coins(keep_copper).each { |amount| withdraw_exact_amount?(amount, settings) } if settings.hometown == hometown
-    DRC.bput('check balance', 'your current balance is', 'You find your jar with little effort', 'Searching methodically through the shelves')
+    case DRC.bput('check balance', 
```

Comment:
IIRC, the `check balance` here was to refresh something in `drinfomon` that already tracked money. How does that differ from this?

Reply:
The benefit here is immediate return, without needing to cross reference which bank / hometown etc.

Reply:
Also I think CharSettings are only accessible to the script that set them (so technically that data is only accessible to drinfomon).